src/DotNetWorker/DotNetWorker.csproj

```diff
@@ -8,7 +8,7 @@
     <AssemblyName>Microsoft.Azure.Functions.Worker</AssemblyName>
     <RootNamespace>Microsoft.Azure.Functions.Worker</RootNamespace>
     <SignAssembly>true</SignAssembly>
-    <DocumentationFile>$(OutputPath)\$(AssemblyName).xml</DocumentationFile>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
Following the advice here -- https://docs.microsoft.com/en-us/dotnet/csharp/codedoc, which I *think* should work for us.

Turns out `$(OutputPath)` isn't always set here -- so you end up with `\....xml`, which resolves to c:\

Reply:
Have you checked the result? It does get packaged, right?

Reply:
Yes, just verified.

Reply:
👍🏼